lib/bitpay/rest_connector.rb

```diff
@@ -99,7 +99,7 @@ def refresh_tokens
     #
 
     def get_token(facade)
-      token = @tokens[facade] || refresh_tokens[facade] || raise(BitPayError, "Not authorized for facade: #{facade}")
+      @tokens[facade.to_sym] || @tokens[facade] || refresh_tokens[facade] || raise(BitPayError, "Not authorized for facade: #{facade}")
```

Comment:
@vijayshreemoyade I understand the  `@tokens[facade.to_sym]` change, but I'm not sure why you removed the variable declaration `token =`

Reply:
@flavioescobar it is a loca variable and required to assign since ruby return last statement.